styles/style.scss

```diff
@@ -0,0 +1,12 @@
+// BASE
+@import "./base/reset";
+@import "./base/root";
+
+// COMPONENTS
+@import "./components/nav";
+@import "./components/button";
+@import "./components/iconSection";
+
+// LAYOUT
+@import "./layout/hero";
+@import "./layout/section";
```

Comment:
As you can see, no code was really changed here. I just moved things into more appropriate files. 

With a site as small as this one is right now, it might not seem like all this extra effort is worth it but it really makes a difference down the road as the site expands

You gotta set yourself up for success!!!

Reply:
Nice! I like it. I knew you could do this with SCSS but never attempted it and this info on how to structure it properly is excellent. Learning to do it the right way.